### ISSUE TITLE
[Accessibility] Icons: Detected by Narrator in Scan Mode, despite role="presentation"?

### ISSUE THREAD

**Issue by HalcyonPride** [OfficeDev/office-ui-fabric-react#8139]
### Environment Information 
 
- __Package version(s)__: Latest
- __Browser and OS versions__: Microsoft Edge, Windows 10 Enterprise, Version 1809, OS build 17763.316

### Please provide a reproduction of the bug in a codepen:
https://codepen.io/anon/pen/aMbyMW
 

 
#### Actual behavior:
 
Narrator stops at Icons in Scan Mode, even though they have `role="presentation"`.
According to the Narrator team:
> The UIA representation (whether it's about `<i>` under the hood or something built around it) claims there is an interactive element. And Narrator stops on that element.

#### Expected behavior:
 
Narrator shouldn't stop on Icons in Scan Mode.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? No
 
Requested priority: Normal; don't think it's a MAS violation
 
Products/sites affected: Anything that uses Fabric Icons

**Comment by leddie24** [comment]
Hi @HalcyonPride, I can't seem to repro this bug.  Can you please provide a screen recording of the bug? Thanks!

**Comment by HalcyonPride** [comment]
@leddie24 said he can repro it now

**Comment by leddie24** [comment]
@micahgodbolt @cschlechty , do you have any recommendations on this issue?  I spoke with @betrue-final-final and he said the best approach would be to have `alt` text attribute similar to an `img` element, where it would read "Image dog jumping through hoop" if you pass in `<img alt="Dog jumping through hoop" />`, but in this case, it would read "Icon CompassNW".  However, we can't simply pass an `alt` attribute or `title` since screen readers aren't aware that the element is an Icon, and simply adding "Icon" to the beginning of an `alt` attribute or `title` doesn't work because we have the issue of localization.

**Comment by cschlechty** [comment]
Take a look at https://css-tricks.com/can-make-icon-system-accessible/ We
likely want to behave similar to calling it a role="img" with an
aria-label. If no label is provided, we could hide it.

**Comment by awalia13** [comment]
Firstly need to identify whether icon is meaningful or not. If meaningful then we can add alt="some meaningful text" so that screen reader will be able to read it in both scan/tab mode. but if icon is not meaningful then we can add alt="" by doing this screen reader will ignore it.

**Comment by cschlechty** [comment]
Icons are not img, and don't support "alt". If meaningful, it should have
and appropriate role and label value. The control consumer will decide if
it's significan.

**Comment by micahgodbolt** [comment]
btw icons can take aria-label no problem, but the role is hard set, so while this does announce properly in scan mode, the role doesn't update
https://codepen.io/micahgodbolt/pen/aMjBRW

Also note, that this still doesn't give us a way to skip the icon, just a way to give it something to read.

In most cases icons on their own will be inside of a button next to text. In those cases narrator only reads the button text, not the icon itself.

**Comment by cschlechty** [comment]
In cases where there is no label, it might make sense to mark them as
hidden to remove them from the UIA tree.

**Comment by micahgodbolt** [comment]
updated with the 2nd icon having 'aria-hidden'. That works great. Should all icons be aria-hidden then? That's the intention right? They are purely decorative.

**Comment by awalia13** [comment]
Agrees with Micah,if they are purely decorative then we can add aria-hidden ="true" so that screen reader will ignore it.

**Comment by micahgodbolt** [comment]
PR is open at #8380 would love some feedback on if this is the correct approach. i.e. Is it better to make `aria-hidden` optional? Would it default to on or off? Or should icons always be hidden?

**Comment by leddie24** [comment]
Closed via #8820
